src/Analysis/GWASV2/GWASV2.css

```diff
@@ -61,3 +61,52 @@
   border-radius: 5px;
   border: 2px solid #dedede;
 }
+
+.GWASV2 .GWASUI-selectionUI button {
+  height: 96px;
+  width: 317px;
+  margin-right: 16px;
+  background: #fafafb;
+  border: 1px solid #2e77b8;
+  border-radius: 4px;
+  transition: all 0.3s cubic-bezier(0.645, 0.045, 0.355, 1);
```

Comment:
can we remove this one?

Reply:
I tested without this and I think it results in a cleaner/calmer UI.

Reply:
Good point! That line of code has been removed.